backend/TreeOfAKind.Infrastructure/Domain/UserProfiles/UserProfileTypeConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using TreeOfAKind.Domain.UserProfiles;
+
+namespace TreeOfAKind.Infrastructure.Domain.Users
+{
+    public class UserProfileTypeConfiguration : IEntityTypeConfiguration<UserProfile>
+    {
+        private const int MaxAuthIdLength = 128;
```

Comment:
Maybe just create some project wise string lengths, like `Long = 2000`, `Very short = 250` etc.

Reply:
Also you have to validate those lenght in the validators.

Reply:
Ok, I was wondering whether abstract those lengths to static classes, but minimizing length values seems wise +1